dspy/adapters/utils.py

```diff
@@ -237,3 +237,14 @@ def _quoted_string_for_literal_type_annotation(s: str) -> str:
     else:
         # Neither => enclose in single quotes
         return f"'{s}'"
+    
+def enumerate_fields(fields: dict) -> str:
```

Comment:
this method has been renamed to `get_field_description_string` in the same file, can we use it? 

I did the renaming because `enumerate_fields` is a bit vague.

Reply:
Good catch, let me use that instead!